app/models/Company.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+use Setting;
+
+use Lang;
+use Sentry;
```

Comment:
I had to remove all three of these `use` statements in order for it to work. The error I was getting when they were still in place was: The use statement with non-compound name 'Setting' has no effect